content/open-source/guides/learn-godot/beginner/index.md

```diff
@@ -0,0 +1,106 @@
+
+## Become a game developer
+
+We collected and reviewed dozens of **free online resources** resources to help you learn to make your first video games with [Godot](https://godotengine.org/). 
+
+This learning path will take you from 0 to your first full-blown games, albeit small ones.
+
+> The journey of a thousand miles begins with one step.
+> 
+> Lao Tzu
+
+Your first game may not be your dream project, but you're going to learn and grow a lot, taking your first steps. You're going to discover a world full of wonders where the further you go, the more creative the process becomes.
+
+> It is good to have an end to journey toward; but it is the journey that matters, in the end.
+> 
+> Ursula K. Le Guin
+
+As you are going to see, there are many great tutorials and tools that you can use to learn game creation from your bedroom for free today.
+
+Now is a great time to learn game development.
+
+## Why Godot? What is Godot?
+
+[Godot](https://godotengine.org/) is a 2D and 3D game engine with a fully-fledged editor, but it is different from the other big game engines out there. It is **Free and Open Source Software**, funded by the community, backed by large organizations like Mozilla or Microsoft, and lead by a team of passionate developers shooting for the stars. 
+
+With **over 1000 contributors** at the time of writing, Godot is becoming more and more competitive even compared to the most significant engines in the games industry.
+
+With separate 2D and 3D engines, an innovative design, it is already packed with a large number of features making 2D game creation fun and rewarding. The team is now working hand-in-hand with the [Vulkan](https://www.khronos.org/vulkan/) representatives to provide cutting-edge 3D in version 4.0.
+
+Godot uses the permissive [MIT](https://opensource.org/licenses/MIT) license. This means there's no limit to what an excellent team of developers can do with it, even for commercial projects.
+
+## Can I make my dream game?
+
+Game creation takes time and effort. Game developers are people who continuously learn new tricks and learn to enjoy the creation process.
+
+I cannot tell whether you will be able to create your dream project or not, depending on how crazy it is. Still, you can undoubtedly make games that people will enjoy, even alone. It will just take some time and dedication.
+
+> Great things are not done by impulse, but by a series of small things brought together.
+> 
+> Vincent Van Gogh
+
+
+## Getting started with game development
+
+Many enthusiasts starting on a journey to learn a new topic or skill **quickly become swamped by the sheer amount of information out there**. Nowadays, the problem isn't access to information as it once was, but rather **too much scattered information**. There's also the problem that most of it **isn't exactly respectable quality**, because anyone can put anything on the internet without any filters and quality control.
```

Comment:
No need for as much bold text. I'd also prefer the tone to be friendly and creative, as a developer talking to a developer, rather than something that sounds a bit like a marketing campaign.

Reply:
I tried rephrasing, let's see if it meets the standard :)